src/tests/e2e/estimatingMonthlyCostUserStoryE2E.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { expect, test } from "@playwright/test";
+import { CalculatorPage } from "../../pageObject/CalculatorPage";
+
+test.describe('e2e', () => {
+    let calculatorPage: CalculatorPage;
+
+    test.beforeEach(async ({page}) => {
+        calculatorPage = new CalculatorPage(page);
```

Comment:
Consider using custom fixtures to provide page objects to test
https://playwright.dev/docs/test-fixtures#creating-a-fixture